### DISCUSSION TITLE
I can't manage to set dir to rtl

### DISCUSSION THREAD

**Discussion by JobSolutions** [bootstrap-email/bootstrap-email#136] [Help]
Hi
I am trying to set the email or part of it to right-to-left direction, but i've been unsuccessful .
Modifying the <body> tag to <body dir="rtl"> turns the whole editor to rtl, still, the direction is not respected in the resulting email.
After doing some tests, i see only <p> respects the direction and right-align

Thanks for your help
Jaime

**Comment by stuyam** [comment]
Can you give me an example of what you are trying and what you are experiencing? Maybe open an issue as well.

**Comment by JobSolutions** [comment]
It's definitely an issue, right-to-left is not respected. Just add `dir="rtl"` to any div, or other elements, and nothing happens.
Also, if i add it to the <body> tag in the online editor, the whole online editor goes RTL (but the generated email is still not rtl).
